app/assets/stylesheets/legacy_sass/sayt/custom.sass

```diff
@@ -340,6 +340,13 @@
     text-align: left
     font-weight: bold
     font-size: smaller
+    /* jQuery UI 1.12 toggle .ui-state-active instead of .ui-state-hover when hovering autocomplete selection
```

Comment:
@viclim can you confirm that an expected result of this change would be the following HTML differences?

<img width="1348" alt="new_type_ahead" src="https://user-images.githubusercontent.com/50927/38204666-d3b440c8-3671-11e8-91bb-e8c3739af7f5.png">
![old_type_ahead](https://user-images.githubusercontent.com/50927/38204667-d3c2fc44-3671-11e8-85ce-6b9e7b9d60d5.png)